build.py

```diff
@@ -588,6 +591,7 @@ def __init__(self, args):
         ctestArgs=context.ctestArgs,
         buildVariant=BuildVariant(context),
         debugPython=("On" if context.debugPython else "Off"),
+        enablePython3=("On" if Python3() else "Off"),
```

Comment:
Simple change to output Python3 state in the output console.

Reply:
@seando-adsk  Correct, just a simple indication in the output console:

![image](https://user-images.githubusercontent.com/48299423/83671101-23ac4980-a5a2-11ea-93cf-6a9db7a601f9.png)

Reply:
I think @seando-adsk's point was that that's indicating that `build.py` itself is running using Python 3, and **not** necessarily indicating that you're building for Python 3. You could run `build.py` using Python 3 but still be building maya-usd for Python 2, in which case that indicator might be misleading.

Reply:
Yes I already realized that :) I agree with you that this could be misleading and in fact in my mind was thinking to change it. something for another day.